### PR TITLE
Gamestats

### PR DESCRIPTION
**Pull request by PhilipDeFraties**
For review: completed lib/game.rb and test/game_test.rb

### CODE REVIEW

File: test/game_test.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+require './test/test_helper'
+require './lib/game.rb'
+
+
+class GamesTest < Minitest::Test
```
Comment (LHJE) [inline test/game_test.rb:5]: I think maybe do GameTest on this line?

**Review by BJSherman80 — APPROVED** [review]
Nice job it looks good to me.

**Review by LHJE — CHANGES_REQUESTED** [review]
Looks so great!  Thanks for this idea!

File: test/game_test.rb (hunk 1 above)
Comment (PhilipDeFraties) [reply test/game_test.rb:5]: Good catch!

**Pull request closed without merging**